### DISCUSSION TITLE
create nominatim database from rail port database directly

### DISCUSSION THREAD

**Discussion by RezaShahbaz** [osm-search/Nominatim#2482] [Q&A]
Hi
I want to create a nominal database directly from the rail port database.
As far as I know, nominitim creates place, planet_osm_nodes, planet_osm_ways, planet_osm_rels tables using the .osm file and osm2pgsql.
If I create these tables from the rail port database, can other tables be created using the .sql functions in the "Nominatim/lib-sql"directory?

**Comment by mtmail** [comment]
Rails port and Nominatim are completely separate systems. They should be installed in separate databases.
